packages/react-heartwood-components/src/components/Modal/components/ModalBody/ModalBody.js

```diff
@@ -10,13 +10,22 @@ type Props = {
 	/** Optional class name */
 	className?: string,
 
+	/** Should the content go to the edge of the modal body? */
+	fullBleed?: boolean,
+
 	/** Optional height of body */
 	height?: string
 }
 
 const ModalBody = (props: Props) => (
 	<div
-		className={cx('modal-body', props.className)}
+		className={cx(
+			'modal-body',
+			{
+				'modal-body__full-bleed': props.fullBleed
```

Comment:
@gcpantazis for modifiers we've typically been using dashes (e.g. `modal-body--full-bleed`)